tests/global_discovery_catalogue/test_gdc_functional.py

```diff
@@ -49,6 +49,16 @@ def gb_client():
     return client
 
 
+@pytest.fixture
+def gdc_broker():
+    options = {
+        'client_id': 'wis2-gdc-test-runner'
```

Comment:
Is it on purpose to define a fix string ? If two clients are started at the same time they will kick each other out.
Depending on the QOS used, it may also have side effect.

Reply:
The context of the GDC tests are such that one client is started, in the context of the GDC orchestrator/runner, for purpose of the tests?